static/js/stream_list.js

```diff
@@ -21,6 +22,11 @@ function active_stream_name() {
 
 function filter_streams_by_search(streams) {
     var search_box = $(".stream-list-filter");
+    if (search_box.length === 0) {
+        // If no stream filter box is there,
+        // return all streams.
+        return streams;
+    }
```

Comment:
In file `frontend_tests.node_tests.stream_list.js`, function `test_sort_pin_to_top_streams` does not add `.stream-list-filter` to body. Hence, `search_box.expectOne()` in line number 31 fails.
So, I added this code if no filter box is detected then return streams without filtering. 
Or I can just append `.stream-list-filter` in body and remove this part of code. Thoughts?